docs/source/examples/wandb_logging.rst

```diff
@@ -0,0 +1,79 @@
+.. _wandb_logging:
+
+===========================
+Logging to Weights & Biases
+===========================
+
+.. customcarditem::
+   :header: Logging to Weights & Biases
+   :card_description: Log metrics and model checkpoints to W&B
+   :image: _static/img/torchtune_workspace.png
+   :link: examples/wandb_logging.html
+   :tags: logging,wandb
+
+
+Torchtune supports logging your training runs to [Weights & Biases](https://wandb.ai).
+
+.. note::
+
+  You will need to install the `wandb`` package to use this feature.
+  You can install it via pip:
+
+  .. code-block:: bash
+
+    pip install wandb
+
```

Comment:
Can you add a "tip" to run `wandb login` before running?

Reply:
good catch